### ISSUE TITLE
Add /sgapeople route to masthead and app.js router

### ISSUE THREAD

**Issue by emily-wilson** [GT-SGA/JacketPages#39]
### NOTE: Wait for the sgapeople branch to merge into react-setup before starting this issue.

In app.js, import the sgapeople component and add a new Route tag to render the react component on that route. Also change all references to the sgapeople.html file to reroute to `/sgapeople` in the masthead file.
